src/menu.js

```diff
@@ -41,6 +41,9 @@ BaseMenu.prototype.events = (function() {
     var events = {};
     var event = UserAgentUtil.isMobile() ? 'tap' : 'click';
     events[event + ' .' + BaseMenu.CLASSES.BODY + ' > li'] = 'handleOptionClick';
+    if (!UserAgentUtil.isMobile()) {
+        events['keyup .' + BaseMenu.CLASSES.BODY + ' > li'] = 'handleOptionClick';
```

Comment:
`UserAgentUtil.isMobile()` and `BaseMenu.CLASSES.BODY` are called/referenced twice in this function. I think it'll be more performant to put their values into local variables and reference the local variables.